### ISSUE TITLE
Make ClientConfig Extra aware of context settings

### ISSUE THREAD

**Issue by bequadrat** [modx-ccc-2015/whishlist#46]
I'd like to use @Mark-H 's great ClientConfig extra in pretty much any project, but unfortunately it's currently not able to set context settings (only system settings afaik). So if you have multiple contexts (like a multi-language website, landing pages outside the main context or complete multiple websites powered by one MODX install), there's no other way to let a client edit specific context settings than to give a technically unexperienced client access to all the context settings. And you don't want that :-)

So maybe we could improve that? It would make the "customer experience" of MODX for end-users with multi-context-sites so much more secure and intuitive.

**Comment by mindeffects** [comment]
Not letting users edit parts of the e.g. settings is a lacking features of MODX itself. By restricting the user to certain namespaces (https://github.com/modx-ccc-2015/whishlist/issues/47), it would be possible to bring those features to MODX itself. @TheBoxer might agree on that?

**Comment by bequadrat** [comment]
Yup, #47 would indeed make ClientConfig a little bit obsolete :-) On the other hand, I suppose extending ClientConfig would be much simpler to achieve as a first step.

**Comment by Mark-H** [comment]
Obsolete? Nahhhh, the interface would still be more appropriate for clients than the settings grid ;)

Will happily sit with some people with UX skills and smart devs to figure out how to implement this from both the user and back-end perspective this weekend.

**Comment by bequadrat** [comment]
Ok, "obsolete" might be the wrong word here. Let's say "less required" :-)
